src/code/z_kankyo.c

```diff
@@ -1209,7 +1243,7 @@ void Environment_Update(PlayState* play, EnvironmentContext* envCtx, LightContex
                     PRINTF("\n" VT_FGCOL(YELLOW) "設定パレット＝[%d] パレット数＝[%d]\n" VT_RST, envCtx->lightSetting,
                            envCtx->numLightSettings);
                 }
-            }
+#endif
```

Comment:
add explicit break? (not even sure if this is the right spot)

Reply:
Adding an explicit break ruins matching in both retail/debug